example/bzlmod/mylib/mylib.cc

```diff
@@ -0,0 +1,12 @@
+#include <fmt/core.h>
+#include <string>
+
+namespace mylib {
+
+int add(int a, int b) { return a + b; }
+
+std::string greet(const std::string &name) {
```

Comment:
Just for my own curiosity, since I haven't written much C++ since 2011: Is `using namespace std` out of favor these days?

Reply:
Out of favor I think. There's a lot of stuff in `std`, and wildcard imports are dangerous: a new C++ release could introduce a naming conflict.